rct229/rulesets/ashrae9012019/section19/section19rule14.py

```diff
@@ -0,0 +1,273 @@
+
+            baseline_modeled_return_as_expected = (
+                True
+                if (is_modeled_with_return_fan_p and return_fans_airflow > ZERO.FLOW)
+                or (
+                    not is_modeled_with_return_fan_p
+                    and return_fans_airflow == ZERO.FLOW
+                )
+                else False
+            )
+
+            baseline_modeled_relief_as_expected = (
+                True
+                if (
+                    (is_modeled_with_relief_fan_p and relief_fans_airflow > ZERO.FLOW)
+                    or (
+                        not is_modeled_with_relief_fan_p
+                        and relief_fans_airflow == ZERO.FLOW
+                    )
+                )
+                else False
+            )
+
+            return {
+                "return_fans_airflow": return_fans_airflow,
+                "relief_fans_airflow": relief_fans_airflow,
+                "baseline_modeled_return_as_expected": baseline_modeled_return_as_expected,
+                "baseline_modeled_relief_as_expected": baseline_modeled_relief_as_expected,
+                "modeled_cfm": modeled_cfm,
+                "supply_minus_OA_flow": supply_minus_OA_flow,
+                "supply_cfm_90_percent": supply_cfm_90_percent,
+            }
+
+        def manual_check_required(self, context, calc_vals=None, data=None):
+            return calc_vals["more_than_one_supply_or_return_fan"]
+
+        def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+
+            return f"{hvac_id_b} has more than one supply or return fan associated with the HVAC system in the baseline and therefore this check could not be conducted for this HVAC sytem. Conduct manual check for compliance with G3.1.2.8.1."
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            return_fans_airflow = calc_vals["return_fans_airflow"]
+            relief_fans_airflow = calc_vals["relief_fans_airflow"]
+            baseline_modeled_return_as_expected = calc_vals[
+                "baseline_modeled_return_as_expected"
+            ]
+            baseline_modeled_relief_as_expected = calc_vals[
+                "baseline_modeled_relief_as_expected"
+            ]
+            modeled_cfm = calc_vals["modeled_cfm"]
+            supply_minus_OA_flow = calc_vals["supply_minus_OA_flow"]
+            supply_cfm_90_percent = calc_vals["supply_cfm_90_percent"]
+
+            more_than_one_supply_or_return_fan = data[
+                "more_than_one_supply_or_return_fan"
+            ]
+            is_modeled_with_return_fan_in_proposed = data[
+                "is_modeled_with_return_fan_in_proposed"
+            ]
+            is_modeled_with_relief_fan_in_proposed = data[
+                "is_modeled_with_relief_fan_in_proposed "
+            ]
+
+            return (
+                not more_than_one_supply_or_return_fan
```

Comment:
I don't think you need this - because if there `more_than_one_supply_or_return_fan`, it would raised in the `manual check`.
In another words, this one will be True all the time.